staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -5110,7 +5110,7 @@ type PodSignature struct {
 // Describe a container image
 type ContainerImage struct {
 	// Names by which this image is known.
-	// e.g. ["k8s.gcr.io/hyperkube:v1.0.7", "dockerhub.io/google_containers/hyperkube:v1.0.7"]
+	// e.g. ["registry.k8s.io/hyperkube:v1.0.7", "dockerhub.io/google_containers/hyperkube:v1.0.7"]
```

Comment:
better if we can use the dummy dns entry https://github.com/kubernetes/k8s.io/pull/3245 to avoid leaking `invalid.registry.k8s.io`

https://github.com/kubernetes/kubernetes/pull/107455

Reply:
We could also use the [`example`](https://en.wikipedia.org/wiki/.example) TLD

```suggestion
	// e.g. ["kubernetes.example/hyperkube:v1.0.7", "cloud-vendor.registry.example/cloud-vendor/hyperkube:v1.0.7"]
```